scripts/googleAppScripts/SheetToGithubIssue.gs

```diff
@@ -6,38 +6,62 @@ function onFormSubmit(e) {
   var category = formResponse['קטגוריה'][0];
   var mainLink = formResponse['לינק לאתר או קישור ראשי'][0];
   var description = formResponse['תיאור פרטי היוזמה'][0];
-  var remarksForAdmins = formResponse['הערות למנהלי העמוד'][0];
   var whatsappLink = formResponse['Whatsapp link'][0];
   var telegramLink = formResponse['Telegram link'][0];
   var driveLink = formResponse['Google Drive link'][0];
-  var gFormLink = formResponse['Google Form link'][0];
-  var gDocLink = formResponse['Google Doc link'][0];
   var discordLink = formResponse['Discord link'][0];
   var instagramLink = formResponse['Instagram link'][0];
   var tiktokLink = formResponse['Tiktok link'][0];
   var twitterLink = formResponse['Twitter / X link'][0];
   var facebookLink = formResponse['Facebook link'][0];
 
-  // Construct the issue body using the form response data
-  var issueBody = "### Initiative Name\n" + initiativeName +
-                    "\n\n### Category\n" + category +
-                    "\n\n### Category\n" + category +
-                    "\n\n### Main Link\n" + mainLink +
-                    "\n\n### Description\n" + description +
-                    "\n\n### WhatsApp Link\n" + (whatsappLink || 'N/A') +
-                    "\n\n### Remarks for Admins\n" + (remarksForAdmins || 'N/A') +
-                    "\n\n### Telegram Link\n" + (telegramLink || 'N/A') +
-                    "\n\n### Drive Link\n" + (driveLink || 'N/A') +
-                    "\n\n### Form Link\n" + (gFormLink || 'N/A') +
-                    "\n\n### Doc Link\n" + (gDocLink || 'N/A') +
-                    "\n\n### Discord Link\n" + (discordLink || 'N/A') +
-                    "\n\n### Instagram Link\n" + (instagramLink || 'N/A') +
-                    "\n\n### TikTok Link\n" + (tiktokLink || 'N/A') +
-                    "\n\n### Twitter/X Link\n" + (twitterLink || 'N/A') +
-                    "\n\n### Facebook Link\n" + (facebookLink || 'N/A');
-
-    // Use the initiative name as the issue title
-    var issueTitle = initiativeName;
+  // todo: adjust the form and add these fields
```

Comment:
Ideally we could get rid of all these, so that changes made in the new initiative yaml and json schema would only be done in the form.

I suggest we do the following. Have the question names be identical to the field titles in the new initiative issue form yaml (English), and put the current (Hebrew) titles in the question description.

Then here just loop over the question names and create the issue title and body from them.
This way, the only field you'd need to specifically know about would be "initiative name" which you'd put in the issue's title (all the rest would be placed via a loop in the body).

Reply:
What is your concern here? You already have the pipeline set up so you can test this change in 5 minutes. And the script is currently only executed manually, so nothing in CI/CD would break by definition...